c-api/objbuffer.po

```diff
@@ -78,7 +78,6 @@ msgstr ""
 "`TypeError` en cas d'erreur."
 
 #: ../Doc/c-api/objbuffer.rst:41
-#, fuzzy
```

Comment:
Le fuzzy ici ne peux pas etre enlevé, car `This function always succeeds.` a été ajouté et doit etre traduit